common/repo/GitRepo.java

```diff
@@ -177,18 +149,12 @@ public void tagHead(String name) {
 
   @Override
   public ImmutableList<File> getUncommittedFiles() {
-    String TWO_WHITE_SPACES = "\\s{2}";
     ImmutableList.Builder<File> files = ImmutableList.builder();
-    CommandResult commandResult = runCommand("status --short");
+    CommandResult commandResult = runCommand("status --porcelain");
     ImmutableList<String> lines = splitLines(commandResult.stdout);
     for (String line : lines) {
-      String[] parts;
-      if (!line.trim().startsWith("M")
-          || !line.trim().startsWith("RM")
-          || !line.trim().startsWith("??")) {
-        line = line.replaceFirst(TWO_WHITE_SPACES, " ");
-      }
-      parts = line.trim().split(" ");
+      line = line.replaceFirst("\\s{2}", " ");
```

Comment:
Would `"  "` work instead of `"\\s{2}"`? It's simpler.

Reply:
Fixed.